DashAI/back/models/parameters/models_schemas/RandomForestClassifier.json

```
@@ -50,8 +50,8 @@
         "max_leaf_nodes": {
             "oneOf": [
                 {
-                    "error_msg": "El parámetro 'max_leaf_nodes' debe ser de tipo entero mayor o igual a 2.",
-                    "description": "El parámetro 'max_leaf_nodes' debe ser de tipo entero mayor o igual a 2.",
+                    "error_msg": "The 'max_leaf_nodes' parameter must be an integer greater than or equal to 2.",
+                    "description": "The 'max_leaf_nodes' parameter must be an integer greater than or equal to 2.",
                     "type": "integer",
```

According to the documentation, this parameter is by default None.
We should also specify this in the error message.
If so, what to use: None, null or empty?

Extra note: Maybe we should also add if the parameters are nullable or not by using `"type": ["int", "null"] `

```suggestion
                    "error_msg": "The 'max_leaf_nodes' parameter must be None or an integer greater than or equal to 2.",
                    "description": "Grow a tree with max_leaf_nodes in best-first fashion.",
                    "type": ["int", "null"]
```

---

PD: I would perhaps eliminate this parameter cause it is complex to understrand.

---

I think it is a good idea to specify when a parameter is nullable, but the frontend would need modifications because it expects the "type" property to be a string.

I think that removing the parameter would have no effect in the frontend, but I'm not sure about the backend.